### PR TITLE
feat: Massburn

### LINKED ISSUE #11201: Massburn
Coming here again with the same request as some time ago, we need mass burn feature. Its not first time someone asking for it.

Can we come up with a simple and quick solution? 

image

Thank you!

### LINKED ISSUE #11209: Delisting on retry burns NFT
### What happened?

https://github.com/user-attachments/assets/4a326ad5-77e5-4b44-bb97-aa287e81cc9d

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [x] Feature
- [x] Refactoring

## Context

- [x] Closes #11201
- [x] Closes #11209
- [x] Removed: Duplicate `burn` token transaction action  code
- [x] Ref: #11123 (added UserCart)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

https://github.com/user-attachments/assets/5aeb60d2-6310-4239-a51a-fe7e364722cb

image

### CODE REVIEW

File: components/common/ItemTransferModal.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    @reset="reset"
+  >
+    <template #body>
+      <hr class="my-4">
+
+      <h2 class="mb-2 font-bold text-text-color capitalize">
+        {{ $t('transaction.transferTo') }}
+      </h2>
+
+      <AddressInput
```
Comment (preschian) [inline components/common/ItemTransferModal.vue:20]: missing `strict` prop? required in this component, or make it optional

**Review by preschian — APPROVED** [review]

**Review by Jarsen136 — APPROVED** [review]

File: components/common/ItemTransferModal.vue (hunk 1 above)
Comment (hassnian) [reply components/common/ItemTransferModal.vue:20]: f30179d6a46163c8160a640d2eb786b799c53971 made it optional . already had a default value

**Comment by exezbcz** [comment]
works for me

**Comment by JustLuuuu** [comment]
Amazing job guys!

**Pull request merged**